### PR TITLE
IGNITE-11386 Actualized configuration of local event listeners.

### PR DESCRIPTION
**Pull request by vsisko**

### CODE REVIEW

File: modules/web-console/frontend/app/configuration/generator/generator/JavaTransformer.service.js [hunk 1]
```diff
@@ -837,6 +847,18 @@ export default class IgniteJavaTransformer extends AbstractTransformer {
                         imports.push(`${grp.class}.${grp.value}`);
                     });
 
+                    break;
+
+                case 'MAP':
+                    if (prop.valClsNameShow === 'ARRAY_EVENTS') {
+                        _.forEach(prop.entries, (lnr) => {
+                            _.forEach(lnr.eventTypes, (type) => {
+                                console.log(type);
```
Comment (nva) [inline modules/web-console/frontend/app/configuration/generator/generator/JavaTransformer.service.js]: console.log should be removed.

File: modules/web-console/frontend/app/configuration/generator/generator/JavaTransformer.service.js [hunk 2]
```diff
@@ -383,6 +383,12 @@ export default class IgniteJavaTransformer extends AbstractTransformer {
     static _toObject(clsName, val) {
         const items = _.isArray(val) ? val : [val];
 
+        if (clsName === 'ARRAY_EVENTS') {
```
Comment (nva) [inline modules/web-console/frontend/app/configuration/generator/generator/JavaTransformer.service.js]: Please rename to EVENTS

File: modules/web-console/frontend/app/configuration/generator/generator/Beans.js [hunk 3]
```diff
@@ -367,8 +367,11 @@ export class Bean extends EmptyBean {
                 keyClsName: dflt.keyClsName,
                 keyField: dflt.keyField || 'name',
                 valClsName: dflt.valClsName,
+                valClsNameShow: dflt.valClsNameShow,
                 valField: dflt.valField || 'value',
-                entries
+                entries,
+                keyClsGenericName: dflt.keyClsGenericName,
```
Comment (nva) [inline modules/web-console/frontend/app/configuration/generator/generator/Beans.js]:
```suggestion
                keyClsGenericType: dflt.keyClsGenericType,
```

File: modules/web-console/frontend/app/configuration/generator/generator/Beans.js [hunk 4]
```diff
@@ -367,8 +367,11 @@ export class Bean extends EmptyBean {
...
+                keyClsGenericExtends: dflt.keyClsGenericExtends
```
Comment (nva) [inline modules/web-console/frontend/app/configuration/generator/generator/Beans.js]:
```suggestion
                isKeyClsGenericTypeExtended: dflt.isKeyClsGenericTypeExtended
```

File: modules/web-console/frontend/app/configuration/generator/generator/JavaTransformer.service.js [hunk 5]
```diff
@@ -514,11 +520,12 @@ export default class IgniteJavaTransformer extends AbstractTransformer {
 
     static _constructMap(sb, map, vars = []) {
         const keyClsName = this.javaTypes.shortClassName(map.keyClsName);
+        const keyClsGeneric = map.keyClsGenericName ? `<${map.keyClsGenericExtends ? '? extends ' : ''}${this.javaTypes.shortClassName(map.keyClsGenericName)}>` : '';
```
Comment (nva) [inline modules/web-console/frontend/app/configuration/generator/generator/JavaTransformer.service.js]: Please try to simplify this expression, may be will be better to separate for two lines or add braces.

File: modules/web-console/frontend/app/configuration/generator/generator/Beans.js (hunk 3 above)
Comment (vsisko) [reply modules/web-console/frontend/app/configuration/generator/generator/Beans.js]: Fields renamed.

File: modules/web-console/frontend/app/configuration/generator/generator/Beans.js (hunk 4 above)
Comment (vsisko) [reply modules/web-console/frontend/app/configuration/generator/generator/Beans.js]: Fields renamed.

File: modules/web-console/frontend/app/configuration/generator/generator/JavaTransformer.service.js (hunk 2 above)
Comment (vsisko) [reply modules/web-console/frontend/app/configuration/generator/generator/JavaTransformer.service.js]: Constant renamed.

File: modules/web-console/frontend/app/configuration/generator/generator/JavaTransformer.service.js (hunk 5 above)
Comment (vsisko) [reply modules/web-console/frontend/app/configuration/generator/generator/JavaTransformer.service.js]: Expression simplified.

File: modules/web-console/frontend/app/configuration/generator/generator/JavaTransformer.service.js (hunk 1 above)
Comment (vsisko) [reply modules/web-console/frontend/app/configuration/generator/generator/JavaTransformer.service.js]: Removed.

**Comment by dspavlov** [comment]
IGNITE-11386 | Resolved
-- | --

**Pull request closed without merging**